apps/condo/domains/organization/integrations/sbbol/sync/index.js

```diff
@@ -10,6 +10,7 @@ const { dvSenderFields } = require('../constants')
 const { syncUser } = require('./syncUser')
 const { syncOrganization } = require('./syncOrganization')
 const { syncSubscriptions } = require('./syncSubscriptions')
+const { syncBankAccounts } = require('./syncClientAccounts')
```

Comment:
Please, use consistent naming
```suggestion
const { syncBankAccounts } = require('./syncBankAccounts')
```